.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ permissions:
 env:
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: 1
+  RUSTFLAGS_aarch64_apple_darwin: "-C linker=cc"
```

Comment:
this overrides `.cargo/config.toml`. Is that the desired behavior here? I really don't have a good way of syncing all these overrides :/